### DISCUSSION TITLE
Se pueden agregar botones extra para hechizos?

### DISCUSSION THREAD

**Discussion by GreatMass** [diasurgical/DevilutionX#5280] [Ideas]
Seria bastante util con el hechicero contar con botones extra para hechizos. Espero sea viable. 
![Screenshot_2022-08-24-17-08-06-297_com adsk sketchbook](https://user-images.githubusercontent.com/112031034/186533224-2f963217-8509-46e9-8831-6f5098977726.png)

**Comment by AJenbo** [comment]
Is that Portugiser?

**Reply by GreatMass** [reply]
Es español (LA)

what I ask is that if it is feasible to add extra buttons for spells.

**Reply by AJenbo** [reply]
Yes, something similar is planned for a future update.

**Reply by GreatMass** [reply]
ok, thanks for your prompt reply. :)
